### ISSUE TITLE
Invoices not generated correctly

### ISSUE THREAD

**Issue by gil-obradors** [irontec/ivozprovider#943]
One invoice number sequence configured before the end of the month, like :
![image](https://user-images.githubusercontent.com/6016929/55330533-90552100-5491-11e9-83ca-4b75e8e94fc1.png)

One invoice scheduler before the end of the month:  ( client PBX ):
![Captura_2019-04-01_15-35-11](https://user-images.githubusercontent.com/6016929/55331580-e034e780-5493-11e9-9feb-36b678c20936.png)

With this log:
```
Apr  1 08:00:04 sip01 [6719]: changelog.INFO: EntityWasCreated > Ivoz\Provider\Domain\Model\Invoice\Invoice#13 > {"inDate":"2019-03-03 23:00:00","outDate":"2019-03-31 21:59:59","taxRate":21,"invoiceTemplateId":1,"brandId":1,"companyId":8,"numberSequenceId":4,"schedulerId
":1,"id":13} [] []
Apr  1 08:00:04 sip01 [6719]: changelog.INFO: EntityWasCreated > Ivoz\Provider\Domain\Model\FixedCostsRelInvoice\FixedCostsRelInvoice#9 > {"quantity":1,"fixedCostId":2,"invoiceId":13,"id":9} [] []
Apr  1 08:00:04 sip01 [6719]: changelog.INFO: EntityWasUpdated > Ivoz\Provider\Domain\Model\InvoiceScheduler\InvoiceScheduler#1 > {"lastExecution":"2019-04-01 06:00:04","lastExecutionError":"","nextExecution":"2019-05-01 06:00:00"} [] []
```

But the finish result:
![Captura_2019-04-01_15-22-17](https://user-images.githubusercontent.com/6016929/55331680-11151c80-5494-11e9-8e8d-ed1b33b1fac9.png)

No billable calls assigned, no invoce generated, and no invoice number sequence generated.
I have assigned manually one invoice number at this generated invoice, but not working.

If I enter to Client view, I can see 2000 registers with prices. I can simulate the cost of the calls.
If I enter to Brand view I can see all billable calls correctly listed.

If I can bring more logs? I have not find where...

Thanks a lot !

**Comment by Kaian** [comment]
Hi @gil-obradors 

I haven't tested this bug yet, but out of curiosity... Are you able to manually create an Invoice for that company in that time period? 

Maybe it provides some kind of error or something. 

Thanks for all the testing!

**Comment by gil-obradors** [comment]
No problem, take your time!

I'm testing with 1 pbx client and 1 residential.
Both the same behavior.
I have tested with different invoice template, all are the default examples ( simple and detailed)
I put a manual invoice number because the system doesn't put automatically.
All no work.

But I must say that one month ago, i make a test and I can generate invoices from other client tests.

The initial installation was from artemis 2.7, now is 2.9.3 .
I haven't changed nothing from code or db, only upgrades.
I can do any test, if you want.

Thanks!
ivozprovider                                  2.9~2.9.3~20190318.1934~33c273a+artemis

**Comment by mmadariaga** [comment]
Hi @gil-obradors ,

I've been able to reproduce the bug, we'll fix it a soon as possible.

Thanks for reporting :)

**Comment by mmadariaga** [comment]
Fixed in #947
